code/modules/projectiles/guns/projectile/shotgun_yw.dm

```diff
@@ -50,4 +50,36 @@
 
 /obj/item/ammo_magazine/m12gdrumjack/empty
 	name = "drum magazine (12 gauge)"
-	initial_ammo = 0
\ No newline at end of file
+	initial_ammo = 0
+
+//scattering shots, old buckshot
+/obj/item/ammo_casing/a12g/scatter
+	name = "scatter shell"
+	desc = "A 12 gauge scattering shell"
+	icon = 'icons/obj/ammo_yw.dmi'
+	icon_state = "scattershell"
+	projectile_type = /obj/item/projectile/scatter/shotgun
+
+/obj/item/projectile/scatter/shotgun
+	name = "Shotgun scatter projectile"
+	spread_submunition_damage = FALSE
+	submunition_spread_max = 60
+	submunition_spread_min = 50
+	submunitions = list(
+		/obj/item/projectile/bullet/shotgun/scatterprojectile = 6
+		)
+/obj/item/projectile/bullet/shotgun/scatterprojectile
+	name = "pellet"
+	fire_sound = 'sound/weapons/Gunshot_shotgun.ogg'
+	damage = 13
+
+/obj/item/weapon/storage/box/scattershot
+	name = "box of shotgun scatter shells"
+	desc = "It has a picture of a gun and several warning symbols on the front.<br>WARNING: Live ammunition. Misuse may result in serious injury or death."
```

Comment:
forgot to close the \<br\> tag here

Reply:
Aint a tag, it just jumps a line, like ingame writing [br]

Reply:
I believe so, copied from the other ammo box desc so i think thats it

Reply:
Yeah br doesn't need an end tag, unlike most other html. It's just a linebreak.

Reply:
I am dumb yeah